### DISCUSSION TITLE
Preview: VTT Sprite Thumbnails

### DISCUSSION THREAD

**Discussion by JadsonLucena** [vidstack/player#673] [Q&A] [answered]
Hello, I would like to know if it is possible to view thumbnails of the VTT file in the progress bar?

Similar to this:
![image](https://user-images.githubusercontent.com/23225809/194865541-3bec2656-0f3a-41ff-aab1-2f697218cae0.png)

**Comment by billygozutok** [comment, marked as answer]
Hey Jadson!

Vidstack doesn't currently support sprite thumbnails. It's a feature we'll definitely be adding in the near future. In the meantime, you check out the  [Slider Video](https://www.vidstack.io/docs/player/components/ui/slider-video/) component.

**Reply by JadsonLucena** [reply]
Thank you very much for the feedback, I will be looking forward to the release of this feature.

I would like to contribute a suggestion:
I noticed that in the designer of this resource, usually url and signed cookies are not taken into account. Thus, even if the resource exists, it is not viable in this scenario.
HLS.js solves this with a solution that I found elegant. He inserted the xhrSetup and fetchSetup callbacks. This allowed us to intercept the requests so that we can add the signatures with path prefix in the internal ulr's of the vtt file or simply add the header "credentials: include" in case of signed cookie.

**Comment by mihar-22** [comment]
Updating here that thumbnails are now supported: https://www.vidstack.io/docs/player/components/sliders/slider-thumbnail 🥳

**Reply by JadsonLucena** [reply]
It was very good, I liked the simplicity in the implementation.

Question: Do you intend to use signed URL or cookie? If yes, do you have any forecasts?
